maven-core/src/main/java/org/apache/maven/internal/impl/DefaultProjectBuilder.java

```diff
@@ -228,9 +228,9 @@ public String getMessage()
                         }
 
                         @Override
-                        public ProjectBuilderProblemSeverity getSeverity()
+                        public BuilderProblemSeverity getSeverity()
```

Comment:
Why isn't it `BuilderProblem.Severity`?

Reply:
No reason, I'll move it as an inner enum.